### PR TITLE
Release version 3.3.0

### PR DESCRIPTION
**Pull request by msujew**
Updates our changelog and versions for the upcoming 3.3.0 release.

### CODE REVIEW

File: packages/langium/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,24 @@
 # Change Log of `langium`
 
+## 3.3.0 (Nov. 2024)
+
```
Comment (spoenemann) [inline packages/langium/CHANGELOG.md:4]: Let's add a link to https://gitlab.eclipse.org/eclipsefdn/emo-team/emo/-/issues/810 and announce the graduation of our project :-)

**Review by spoenemann — APPROVED** [review]

File: packages/langium/CHANGELOG.md (hunk 1 above)
Comment (msujew) [reply packages/langium/CHANGELOG.md:4]: Good idea!

**Comment by cdietrich** [comment]
@msujew will you release the new vscode extension as well to have the (new grammar) validations in sync?

**Comment by msujew** [comment]
@cdietrich Thanks for the hint, they're online :)

**Pull request merged**